SampleApp/app/src/main/java/one/zoop/gatewaysdk/sampleapp/MainActivity.java

```diff
@@ -52,14 +58,14 @@ private void uiInit() {
         etGatewayId = findViewById(R.id.etGatewayId);
         btItr = findViewById(R.id.btItr);
         tvResult = findViewById(R.id.tvResponse);
-        environment = "QT_PP";
+        environment = "QT_T";
```

Comment:
@divyank96 should this be PP?

Reply:
Yes I was testing on test environment forgot to change